frontend/app/routes/_gcweb-app.personal-information._index.tsx

```diff
@@ -53,7 +54,7 @@ export default function PersonalInformationIndex() {
           title={t('personal-information:index.preferred-language')}
           icon="glyphicon-globe"
         >
-          {languageName}
+          {getNameByLanguage(i18n.language, preferredLanguage)}
```

Comment:
Use [Logical AND operator (&&)](https://react.dev/learn/conditional-rendering#logical-and-operator-)
```suggestion
          {preferredLanguage && getNameByLanguage(i18n.language, preferredLanguage)}
```

Reply:
Fixed it to be able to close the user story.